src/main/java/me/zeroeightsix/kami/module/modules/render/MobOwner.kt

```diff
@@ -12,27 +12,25 @@ import kotlin.math.pow
 
 /**
  * I see you also watch FitMC :eyes:
- *
- * @author cookiedragon234
  * Taken from Backdoored 1.8.2 source
- *
- *
- * UUID to username method and caching methods added by dominikaaaa
```

Comment:
if you remove the UUID note (like 70% of the actual module), you can remove the entire comment

Reply:
So do I add it back or what

Reply:
No, remove the entire comment